### DISCUSSION TITLE
Version 2.0

### DISCUSSION THREAD

**Discussion by benalexb** [code-chat-br/whatsapp-api#134] [Q&A]
Hello everyone,

I've been doing some testing using this project and so far I'm very impressed by your work! One question about the api version though. The docs mention a version 2.0 of the api, but I can't seem to find it anywhere. The latest tag is 1.3, which is also the version I have been testing.

If there is such a thing as a version 2.0, where can I find it?

In advance, thank you so much for any information you are able to share!

Sincerely,
Ben

**Comment by jrCleber** [comment]
Version **2.0** is in a private repository and a membership fee is charged to be included in the repository.

See the version 2 documentation [here](https://docs.codechat.dev/).

**Comment by benalexb** [comment]
@jrCleber I would love to join! I was going through the documentation you shared but I only see information on how to use v2. Who can I get in touch with in order to discuss the possibility of becoming a paid member? I would also love to contribute to the code base if possible 🙏

**Reply by jrCleber** [reply]
V2 is multi-company.
In other words, you can create a business and link countless instances to that business.

* There is the possibility of working with three databases: MongoDb, MySql/MariaDb and PostgreSQL to save data for each instance.
* New ways to send messages, advanced chat and group controls.
* Request returns are better prepared.
* There is the possibility of creating multiple webhooks per instance.
* The data sent by the webhook is more detailed.
* The project infrastructure is cleaner and more organized.
* Writing the code made it process faster.
* Integration with chatwoot through n8n is being developed.
* Instance management dashboard is being developed.
* Integration with MinIO and AWS S3 for storage and media management.

The cost of membership to the private repository is $115 for a period of one year.

---

Payment can be made via payment link.
After payment confirmation, the name **benalexb** will be included as a project collaborator and will have access to the code and its updates.

**Reply by benalexb** [reply]
sounds like a plan! What's the proper channel for us to move forward?

**Reply by jrCleber** [reply]
Friend, good morning.

Payment can be made through this [link](https://buy.stripe.com/cN29DF80k5zhfmg3ci).


After payment confirmation, send me confirmation via this [contact](https://wa.me/message/SOHIKF5G3BS6M1?src=qr) and I will include it in the repository.

<img width="25%" src="https://github.com/code-chat-br/whatsapp-api/assets/65574896/d2562b6d-5a88-4348-8a8a-2ba3211a047a">
